integration/simulation/network/inmemory.go

```diff
@@ -36,6 +36,7 @@ func (n *basicNetworkOfInMemoryNodes) Create(params *params.SimParams, stats *st
 	n.ethNodes = make([]*ethereum_mock.Node, params.NumberOfNodes)
 	obscuroNodes := make([]host.MockHost, params.NumberOfNodes)
 	n.obscuroClients = make([]rpcclientlib.Client, params.NumberOfNodes)
+	p2pLayers := make([]*p2p.MockP2P, params.NumberOfNodes)
```

Comment:
I don't really understand the name `layers`, not sure what the word should be though, something like p2pConnections or p2pAccessor perhaps?

Reply:
I don't want connections, as these aren't connections, they're ways to make the connections.

Think accessors is too meaningless.

I used layers because this is the P2P layer of the node that's used for communication.

Reply:
Fair, maybe it's the pluralisation throwing me. I'm picturing layers and I'm picturing p2p connections and struggling to marry those images up in mind but it's just one 'layer' per node.

Reply:
Let's call it `p2psLayer` instead :D

Reply:
Don't hate the p2pLayer, hate the p2gAme